modules/gapi/src/backends/onnx/gonnxbackend.cpp

```diff
@@ -256,6 +264,29 @@ inline void preprocess(const cv::Mat& src,
     }
 }
 
+inline void preprocess(const std::unique_ptr<cv::MediaFrame::View>& view,
+                       const cv::GFrameDesc& desc,
+                       const cv::gimpl::onnx::TensorInfo& ti,
+                             cv::Mat& dst,
```

Comment:
Can `dst` be the last argument here? I believe that would align with our other API's better (kernels for example)

Reply:
Sure. Fixed.